gempak/ush/gfs_meta_mar_comp.sh

```diff
@@ -1,158 +1,154 @@
-#! /bin/sh
-# Metafile Script : gfs_meta_mar_comp.sh
+#! /usr/bin/env bash
 #
-# This is a script which creates a metafile that runs a comparison of 500 MB 
-# heights and PMSL between the older GFS model run and the newer one. The 
-# metafile also generates a comparison between the UKMET older run and the newer
-# GFS model run.
+# Metafile Script : gfs_meta_mar_comp.sh
 #
-# Log :
-# J. Carr/PMB    12/07/2004    Pushed into production.
-
 # Set up Local Variables
 #
-set -x
-#
-export PS4='MAR_COMP_F${fend}:$SECONDS + '
-rm -Rf $DATA/GEMPAK_META_MAR
-mkdir -p -m 775 $DATA/GEMPAK_META_MAR $DATA/MAR_COMP
 
-cd $DATA/MAR_COMP
-cp ${HOMEgfs}/gempak/fix/datatype.tbl datatype.tbl
+ource "${HOMEgfs}/ush/preamble.sh"
```

Comment:
Missing "s". Should be:
`source "${HOMEgfs}/ush/preamble.sh"`

Reply:
Fixed